### PR TITLE
translated 2020/performance to Dutch

### PR DESCRIPTION
**Pull request by noah-vdv**
Makes progress on #1750

### CODE REVIEW

File: src/content/nl/2020/performance.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+
+De keuze van [hostingprovider](https://ismyhostfastyet.com/) of [CDN](https://www.cdnperf.com/) zal een trapsgewijs effect hebben op de snelheid. Het nemen van deze beslissingen op basis van de snelst mogelijke levering zal helpen bij het verbeteren van FCP en TTFB, vooral op langzamere netwerken. FCP wordt ook aanzienlijk beïnvloed door de laadtijd van lettertypen, dus [ervoor zorgen dat tekst zichtbaar is terwijl weblettertypen worden gedownload](https://web.dev/font-display/) is ook een waardevolle strategie (vooral waar bij langzamere verbindingen deze bronnen kostbaar zijn om te halen).
+
+Als we naar de "offline" statistieken kijken, kunnen we afleiden dat een aanzienlijk aantal FCP-problemen ook _niet_ gecorreleerd zijn met het netwerktype. We zien geen significante winsten in deze categorie, wat we wel zouden als die bewering waar was. Het lijkt erop dat de weergave niet zozeer wordt vertraagd door het ophalen van JavaScript, maar wordt beïnvloed door het parseren en uitvoeren.
+
+## Time to First Byte
```
Comment (bazzadp) [inline src/content/nl/2020/performance.md:311]:
```suggestion
## <span lang="en">Time to First Byte</span> {time-to-first-byte}
```

File: src/content/nl/2020/performance.md [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+
+Time to First Byte (TTFB) is de tijd die nodig is vanaf het initiële HTML-verzoek totdat de eerste byte terugkomt in de browser. Problemen met het snel verwerken van verzoeken kunnen snel overlopen in andere prestatiestatistieken, omdat ze niet alleen verven maar ook het ophalen van bronnen vertragen.
```
Comment (bazzadp) [inline src/content/nl/2020/performance.md:313]:
```suggestion
<span lang="en">Time to First Byte</span> (TTFB) is de tijd die nodig is vanaf het initiële HTML-verzoek totdat de eerste byte terugkomt in de browser. Problemen met het snel verwerken van verzoeken kunnen snel overlopen in andere prestatiestatistieken, omdat ze niet alleen verven maar ook het ophalen van bronnen vertragen.
```

File: src/content/nl/2020/performance.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+
+Kijkend naar de [mobiele snelheden wereldwijd voor december 2018-november 2019](https://www.speedtest.net/insights/blog/content/images/2020/02/Ookla_Mobile-Speeds-Poster_2020.png), kunnen we zien dat wereldwijd mobiele verbindingen niet supersnel zijn. Die netwerksnelheden en technologiestandaarden voor mobiele netwerken (zoals 5G) zijn niet gelijkmatig verdeeld en beïnvloeden TTFB. Zie als voorbeeld [deze kaart met netwerken in Nigeria](https://www.mobilecoveragemaps.com/map_ng#7/8.744/7.670) - het grootste deel van het land heeft 2G- en 3G-dekking, met weinig 4G-bereik.
+
+Wat verrassend is, is het relatief evenveel goede TTFB-resultaat tussen offline en 4G-bronnen. Met Service Workers konden we verwachten dat sommige van de TTFB-problemen zouden worden beperkt, maar die trend wordt niet weerspiegeld in de bovenstaande grafiek.
+
+## Performance Observer gebruik
```
Comment (bazzadp) [inline src/content/nl/2020/performance.md:363]:
```suggestion
## <span lang="en">Performance Observer</span> gebruik {performance-observer-gebruik}
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
Looks pretty good but missed some spans.

**Comment by noah-vdv** [comment]
Wow that's a lot them, sorry I'll definitely add those on the other files. Also what do the { } next to headers add?

**Comment by bazzadp** [comment]
> Wow that's a lot them, sorry I'll definitely add those on the other files. Also what do the { } next to headers add?

Each header is also a link. For example: https://almanac.httparchive.org/en/2020/performance#core-web-vitals-largest-contentful-paint

The link anchor id is auto generated based on the contents so when you add the spans you get an untidy id. The `{ }` allows you to specify an id to override this rather than have the untidy id.

**Review by bazzadp — APPROVED** [review]

**Comment by bazzadp** [comment]
Thank you! Super speedy and top quality effort!

**Pull request merged**